jjb/edgex-templates-performance.yaml

```diff
@@ -0,0 +1,227 @@
+
+    branch: master
+    submodule-recursive: true
+    pre_build_script: ''
+    build_script: ''
+    post_build_script: ''
+    status-context: ''
+    workspace: '$HOME/$BUILD_ID/gopath/src/github.com/edgexfoundry/{project-name}/'
+    #####################
+    # Job Configuration #
+    #####################
+
+    properties:
+      - lf-infra-properties:
+          project: '{project}'
+          build-days-to-keep: '{build-days-to-keep}'
+      - github:
+          url: '{git-url}/{github-org}/{project}'
+
+    parameters:
+      - string:
+          name: PROJECT
+          default: '{project}'
+          description: |
+              Parameter to identify a SCM project to build. This is typically
+              the project repo path. For example: ofextensions/circuitsw
+      - string:
+          name: STREAM
+          default: '{stream}'
+          description: |
+              Stream is often set to the same name as 'branch' but can
+              sometimes be used as a name representing a project's release code
+              name.
+      - string:
+          name: INFLUXDB
+          default: '{influxDBHost}'
+          description: |
+              InfluxDB hostname or ip address, can't be empty
+      - string:
+          name: NODE_EDGEX_1
+          default: '{node1}'
+          description: |
+              First edgeX Services run on this node, can't be empty
+      - string:
+          name: NODE_EDGEX_2
+          default: ''
+          description: |
+              Second edgeX Services run on this node. Set default to blank, 
+              because there is only 3 testcases for now.
+      - string:
+          name: NODE_EDGEX_3
+          default: ''
+          description: |
+              Second edgeX Services run on this node. Set default to blank, 
+              because there is only 3 testcases for now.
+      - string:
+          name: NODE_JMETER
+          default: '{node4}'
+          description: |
+              Trigger JMeter script, can't be empty
+
+
+- performance_verify_boiler_plate: &performance_verify_boiler_plate
+    name: performance_verify_boiler_plate
+
+    scm:
+      - lf-infra-github-scm:
+          url: '{git-clone-url}{github-org}/{project}'
+          refspec: '+refs/pull/*:refs/remotes/origin/pr/*'
+          branch: '$sha1'
+          submodule-recursive: '{submodule-recursive}'
+          choosing-strategy: default
+          jenkins-ssh-credential: '{jenkins-ssh-credential}'
+          submodule-timeout: 10
+
+- performance_testing_boiler_plate: &performance_testing_boiler_plate
+    name: performance_testing_boiler_plate
+
+    scm:
+      - lf-infra-github-scm:
+          url: '{git-clone-url}{github-org}/{project}'
+          refspec: ''
+          branch: 'refs/heads/{branch}'
+          submodule-recursive: '{submodule-recursive}'
+          choosing-strategy: default
+          jenkins-ssh-credential: '{jenkins-ssh-credential}'
+          submodule-timeout: 10
+
+    #triggers:
```

Comment:
Is the plan to remove all of this commented-out code once testing is complete, before merging? (this applies to the several sections below as well; I'll just put the comment here)

Reply:
Yes, I will remove the commented code after testing is complete on sandbox. Thanks.